overview/xarray-in-45-min.ipynb

```diff
@@ -36,23 +36,13 @@
     "%config InlineBackend.figure_format='retina'"
```

Comment:
<p>"See the <a href="https://xarray.pydata.org/en/stable/indexing.html" target="_blank">documentation</a> for more"</p><p> </p><p>There's a bunch more places we could do this. I don't know why I didn't do it the first time around.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>